transformations/multi_pivot_paraphrases_generation/requirements.txt

```diff
@@ -0,0 +1,6 @@
+EasyNMT
+numpy
+scikit-learn
+tensorflow
+tensorflow-hub
+transformers
```

Comment:
I think most of these libraries are present in the main requirements.txt file. You might want to skip adding them.

Reply:
you are right